frontend/assets/css/prime.scss

```diff
@@ -453,10 +453,10 @@
   border: 1px solid var(--input-border-color);
   color: var(--input-active-text-color);
   border-radius: 0 0 var(--border-radius) var(--border-radius);
+  transform: translateY(-1px);
```

Comment:
e.g. this is what the warning was looking like
![sass-deprecation](https://github.com/user-attachments/assets/ead0e09f-35a0-437b-89ec-8c3257345726)